test/old/testdir/test_spellfile.vim

```diff
@@ -1083,4 +1083,4 @@ func Test_mkspell_empty_dic()
 endfunc
 
 
-" vim: shiftwidth=2 sts=2 expandtab
+" vim: shiftwidth=2 sts=2 expandtab
```

Comment:
Missing trailing EOL

Reply:
Out of curiosity: shouldn't StyLua take care of that? Asking because maybe there is something wrong in my setup and it didn't…

Reply:
This is a vimscript file, not a Lua file...

Reply:
Oh, brain fart. My bad. Sorry hahaha…

Reply:
Fixed in 65934874a ✨